native/Avalonia.Native/src/OSX/AvnView.mm

```diff
@@ -207,7 +208,11 @@ - (void) viewDidChangeBackingProperties
 
     if(_parent != nullptr)
     {
-        _parent->BaseEvents->ScalingChanged([_parent->Window backingScaleFactor]);
+        double scaling = 1;
+        
+        _parent->GetScaling(&scaling);
```

Comment:
I wonder if we can just use https://developer.apple.com/documentation/appkit/nsview/1483301-window?language=objc instead of asking the C++ part. With embedding scenarios we won't have the window instance provided anyway.

I don't remember, however, why we weren't using it before.

Reply:
Hmm y that should work. That way we can implement GetScaling at the TopLevelImpl

Reply:
Make sure to add a null check though, since the view might not be connected to anything at the point.